static/js/publisher/release/releasesController.js

```diff
@@ -387,24 +335,16 @@ class ReleasesController extends Component {
 
   handleCloseResponse(json, channels) {
     if (json.success) {
-      this.setState(state => {
-        const { releasedChannels } = state;
-
-        if (json.closed_channels && json.closed_channels.length > 0) {
-          json.closed_channels.forEach(channel => {
-            // make sure default channels get prefixed with 'latest'
-            if (channel.indexOf("/") === -1) {
-              channel = `latest/${channel}`;
-            }
-
-            delete releasedChannels[channel];
-          });
-        }
+      if (json.closed_channels && json.closed_channels.length > 0) {
+        json.closed_channels.forEach(channel => {
+          // make sure default channels get prefixed with 'latest'
```

Comment:
not always true :s! Node for example the default isn't "latest"

We need to figure out a way to expose the 'default' throughout the app.

Reply:
@Lukewh I think (and hope) that this is a bit different meaning of 'default'. Why node is on different track by default in the store (they don't have latest) channel map API reports it fully as "10/stable" not as "stable".

The code here is for the case when just an risk name is used by API which I believe is only a case for latest track. So channel "beta" is equivalent of "latest/beta" always. But I will verify that.

Reply:
Yes. Looking at the channel map data for node it seems that channels in (default) track "10" are named fully as "10/stable" and channels in track "latest" use the short form of "stable".

```
"10": [{"channel": "10/stable" ...
"latest": [{"channel": "edge",  ...
```

Reply:
I'll update the comment not to use the word 'default' which indeed may be confused with other meaning of "default track" the snap uses.